### DISCUSSION TITLE
Option `flat` missing in [@types/canvas-confetti]

### DISCUSSION THREAD

**Discussion by frederikheld** [DefinitelyTyped/DefinitelyTyped#71328] [Issues with a @types package]
The declaration for `confetti` is missing the option `flat`.

It could be added like this:

```typescript
declare namespace confetti {
// [...]

  /**
   * Optionally turns off the tilt and wobble that three dimensional confetti would have in the real world. Yeah, they look a little sad, but y'all asked for them, so don't blame me.
   * @default false
   */
  flat?: boolean | undefined;
// [...]
}
```

This option was introduced in PR https://github.com/catdad/canvas-confetti/pull/202

I took the description from the canvas-confetti README.

I tried to send a PR instead of this issue, but I did not understand how the tests for this package are structured and how I would correctly test this addition without breaking any existing tests.

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
I just opened a PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71442 to address this. You may want to take a look.

**Comment by frederikheld** [comment]
Thanks for taking quick action on this one :-)

Just out of curiosity: why are you using .9999 as fix version and why are you jumping from 1.6.9999 to 1.9.9999 with this PR?

**Reply by hkleungai** [reply]
Hi, I think using `.9999` is how DefinitelyTyped do semver. You may want to check [this readme section](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library).
